mne/source_estimate.py

```diff
@@ -1395,7 +1419,10 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
                                       config_opts=config_opts,
                                       subjects_dir=subjects_dir, figure=figure,
                                       views=views, colorbar=colorbar,
-                                      clim=clim)
+                                      clim=clim, cortex=cortex, size=size,
+                                      background=background,
+                                      foreground=foreground,
+                                      time_unit=time_unit)
```

Comment:
@Eric89GXL do I understand correctly that you are +1 for adding an `initial_time_shown` parameter to `Brain.add_data()`?